tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/OpenTelemetry/TracerProviderBuilderIntegration.cs

```diff
@@ -99,13 +100,13 @@ static TracerProviderBuilderIntegration()
             ilWriter.EmitCall(OpCodes.Call, targetAddProcessorMethod, null);
             ilWriter.Emit(OpCodes.Ret);
 
-            return (Func<object, object, object>)dynMethod.CreateDelegate(typeof(Func<object, object, object>));
+            return (Func<object?, object?, object?>)dynMethod.CreateDelegate(typeof(Func<object?, object?, object?>));
```

Comment:
I don't think we need (or should have) the nulls here :thinking: the  builderType and baseProcessorOfActivityType are both expecting non-null etc, as is the callee, so I don't think we should expect non-null values?